cfbs/cfbs_config.py

```diff
@@ -98,6 +99,39 @@ def save(self):
     def longest_module_name(self) -> int:
         return max((len(m["name"]) for m in self["build"]))
 
+    def _handle_input(self, input):
```

Comment:
I would actually move this into `cfbs.prompts` and drop the first commit because it would eliminate the circular imports.

Reply:
Oh, I lost track of the changes in those files! I thought `CFBSConfig()` was a config not something that actually even adds and removes modules etc. Anyway, I still think this input handling should be separated. If for nothing else then at least for better reusability.